spec/index.bs

```diff
@@ -11137,8 +11232,8 @@ enum GPUStoreOp {
 
 #### Render Pass Layout #### {#render-pass-layout}
 
-{{GPURenderPassLayout}} contains the layout of the render targets for the current pass,
-which determines the compatibility of the pass with render pipelines.
+{{GPURenderPassLayout}} contains the layout of the render targets for the current pass or render
+bundle, which determines the compatibility of the pass or bundle with render pipelines.
```

Comment:
Also the compatibility between passes and bundles.
Actually, technically this dictionary is only publicly exposed for render bundles. Render passes have one internally, but they derive it from GPURenderPassDescriptor. Since this bit of text is developer-targeted maybe it should clarify this. WDYT:
```suggestion
{{GPURenderPassLayout}} declares the layout of the render targets of a {{GPURenderBundle}}.
It is also used internally to describe
{{GPURenderPass}} [$derive render targets layout from pass|layouts$] and
{{GPURenderPipeline}} [$derive render targets layout from pipeline|layouts$].
It determines compatibility between render passes, render bundles, and render pipelines.
```

Reply:
Sounds perfect!